src/python/pants/backend/project_info/tasks/export_dep_as_jar.py

```diff
@@ -90,6 +90,7 @@ def prepare(cls, options, round_manager):
         round_manager.require_data("zinc_args")
         round_manager.require_data("runtime_classpath")
         round_manager.require_data("export_dep_as_jar_signal")
+        round_manager.require_data("jvm_modulizable_targets")
```

Comment:
If we are using `jvm_modulizable_targets` to signal that we are exporting then we don't need the `'export_dep_as-jar_signal` product anymore.

Reply:
Good point! Adjusted.